dcat/index.html

```diff
@@ -895,25 +899,43 @@ <h4>Property: themes</h4>
 			
         </section>
 
+<!-- DCAT2
         <section id="Property:catalog_has_part">
+		
             <h4>Property: has part</h4>
-
             <aside class="note">
               <p>
                 Property added in this context in DCAT 2.
               </p>
             </aside>
+-->
+        <section id="Property:catalog_resource">
+		
+            <h4>Property: resource</h4>
+            <aside class="note">
+              <p>
+                Property added in this context in DCAT 3.
+              </p>
+            </aside>
 
 
             <table class="def">
+<!-- DCAT2			
                 <tr><th>RDF Property:</th><td><a href="http://purl.org/dc/terms/hasPart"><code>dcterms:hasPart</code></a></td></tr>
+-->				
+                <tr><th>RDF Property:</th><td><a href="http://www.w3.org/ns/dcat#resource"><code>dcat:resource</code></a></td></tr>
+
                 <tr><th class="prop">Definition:</th><td>A resource that is listed in the catalog.</td></tr>
-<!--                    
+                    
                 <tr><th class="prop">Domain:</th><td><a href="#Class:Catalog"><code>dcat:Catalog</code></a></td></tr>
                 <tr><th class="prop">Range:</th><td><a href="#Class:Resource"><code>dcat:Resource</code></a></td></tr>
--->
+
                 <tr><th class="prop">Usage note:</th><td>This is the most general predicate for membership of a catalog. Use of a more specific sub-property is recommended when available. </td></tr>
+<!-- DCAT2
                 <tr><th class="prop">See also:</th><td>Sub-properties of <code>dcterms:hasPart</code> in particular <a href="#Property:catalog_dataset"><code>dcat:dataset</code></a>, <a href="#Property:catalog_catalog"><code>dcat:catalog</code></a>, <a href="#Property:catalog_service"><code>dcat:service</code></a>. </td></tr>
+-->				
+                <tr><th class="prop">See also:</th><td>Sub-properties of <code>dcat:resource</code> in particular <a href="#Property:catalog_dataset"><code>dcat:dataset</code></a>, <a href="#Property:catalog_catalog"><code>dcat:catalog</code></a>, <a href="#Property:catalog_service"><code>dcat:service</code></a>. </td></tr>
+				
```

Comment:
Is there an explicit relationship from `dcat:resource` to `dcterms:hasPart` ?

Reply:
Thanks for spotting this, @dr-shorthair . I forgot to add the row about the fact that `dcat:resource` is a sub-property of `dcterms:hasPart`. Now fixed via commit https://github.com/w3c/dxwg/pull/1470/commits/d587d4ed8fe60fa50982a1ed4ce4570871398ab4